web/ui/templates/_base.html

```diff
@@ -40,8 +40,9 @@
             <li><a href="{{ pathPrefix }}/alerts">Alerts</a></li>
             <li><a href="{{ pathPrefix }}/graph">Graph</a></li>
             <li><a href="{{ pathPrefix }}/status">Status</a></li>
+            <li><a href="{{ pathPrefix }}/config">Config</a></li>
```

Comment:
I'm seldom interested in the configuration, as that doesn't change often. I check the target table often when I have to debug why someone's targets are not working. Linking them only from the config page is an unnecessary extra step.

I read your statement as the wish to have a direct link to a particular job. We can achieve that with a dropdown menu https://getbootstrap.com/examples/navbar/.